src/app/content/components/Attribution.tsx

```diff
@@ -161,8 +161,12 @@ class Attribution extends Component<Props> {
 
     const seniorAuthors = book.authors.filter((author) => author.value.senior_author);
 
+    const authorsToDisplay = seniorAuthors.length > 0 ?
+      seniorAuthors.map(({value: {name}}) => name).join(', ')
+      : book.authors.slice(0, 2).map(({value: {name}}) => name).join(', ');
```

Comment:
you could leave the map below where it was and not have it here twice. the logic would reduce to 
```
const authorsToDisplay = seniorAuthors.length > 0
  ? seniorAuthors
  : book.authors.slice(0, 2)
```